gravitee-am-ui/src/app/domain/applications/application/advanced/factors/mfa-challenge/risk-based/mfa-risk-based.component.html

```diff
@@ -15,26 +15,30 @@
     limitations under the License.
 
 -->
-
+<info-banner
```

Comment:
**question**: When I click on the "More Info" button a new tab is open to see the details but the application settings are saved when we click on this button. Is it the right behaviour ? I have some concerns about this behaviour as if the application settings are not defined but not finalized, the save action will redeploy the application on the gateway with unexpected settings... this is an issue IMO.

Reply:
fixed